desktop-widgets/usermanual.cpp

```diff
@@ -35,6 +35,27 @@ void SearchBar::enableButtons(const QString &s)
 	ui.findNext->setEnabled(s.length());
 }
 
+#ifdef USE_WEBENGINE
+MyQWebEnginePage::MyQWebEnginePage(QObject* parent) : QWebEnginePage(parent)
```

Comment:
```suggestion
WebEnginePage::WebEnginePage(QObject* parent) : QWebEnginePage(parent)
```

naming is consistent with other inherited object cases.